hadoop-pipeline-application/src/main/java/org/molgenis/hadoop/pipeline/application/processes/InStreamHandler.java

```diff
@@ -0,0 +1,49 @@
+package org.molgenis.hadoop.pipeline.application.processes;
+
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+
+/**
+ * Handles the streams retrieved from the process.
+ */
+public class InStreamHandler extends StreamHandler
+{
+	/**
+	 * Stores the write stream to be used.
+	 */
+	InputStream processStream;
+
+	/**
+	 * Object to which the output stream will be written to.
+	 */
+	InContainer inContainer;
+
```

Comment:
Update: `processStream` is now private and `inContainer` is removed. The `InContainer` is now defined in the subclasses instead as subclasses have their own `InContainer` implementation and therefore keeping it generic would require multiple class castings (between the generic storage version and specific version when using it's unique methods).